nav2_bt_navigator/src/navigators/navigate_to_pose.cpp

```diff
@@ -212,6 +211,8 @@ NavigateToPoseNavigator::initializeGoalPose(ActionT::Goal::ConstSharedPtr goal)
   start_time_ = clock_->now();
   auto blackboard = bt_action_server_->getBlackboard();
   blackboard->set<int>("number_recoveries", 0);  // NOLINT
+  blackboard->set<int>("follow_path_error_code", 0);
```

Comment:
Set based on message NONE types. I know they're `0` but that removes a magic number from the code + if it changes in the future, we don't have a subtle breakage